src/bankid.ts

```diff
@@ -266,38 +266,42 @@ export class BankIdClient {
         : undefined,
     };
 
-    return this._call<SignRequest, SignResponse>(BankIdMethod.sign, parameters);
+    return this.#call<SignRequest, SignResponse>(BankIdMethod.sign, parameters);
   }
 
   collect(parameters: CollectRequest) {
-    return this._call<CollectRequest, CollectResponse>(
+    return this.#call<CollectRequest, CollectResponse>(
       BankIdMethod.collect,
       parameters,
     );
   }
 
   cancel(parameters: CollectRequest): Promise<CancelResponse> {
-    return this._call<CollectRequest, CancelResponse>(
+    return this.#call<CollectRequest, CancelResponse>(
       BankIdMethod.cancel,
       parameters,
     );
   }
 
+  _awaitPendingCollect(orderRef: string) {
+    console.warn("This method has been renamed to 'awaitPendingCollect");
+    this.awaitPendingCollect(orderRef);
+  }
+
   async authenticateAndCollect(
     parameters: AuthRequest,
   ): Promise<CollectResponse> {
     const authResponse = await this.authenticate(parameters);
-
-    return this._awaitPendingCollect(authResponse.orderRef);
+    return this.awaitPendingCollect(authResponse.orderRef);
   }
 
   async signAndCollect(parameters: SignRequest): Promise<CollectResponse> {
     const signResponse = await this.sign(parameters);
 
-    return this._awaitPendingCollect(signResponse.orderRef);
+    return this.awaitPendingCollect(signResponse.orderRef);
   }
 
-  _awaitPendingCollect(orderRef: string): Promise<CollectResponse> {
+  awaitPendingCollect(orderRef: string): Promise<CollectResponse> {
```

Comment:
I don't particularly like the name of this method tbh ... it does more of a `poll` 🤔 `waitUntilCollectSuccess` or something similar?

Reply:
Yeah agree.. 
Tbh not sure if this method/pattern will be relevant when we adding the 6.x changes ... 
Initiating a session with a fixed PNO will be removed for example, so starting ánd collecting makes no sense anymore; it would always need the user to act on something before starting the collection process. With the PR containing these changes, we could re-evaluate this method?